src/seedu/addressbook/AddressBook.java

```diff
@@ -202,13 +202,21 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        startUserInputLoop();
```

Comment:
- consider a different name as discussed in the tutorial.
- you have a system that is 
 1. welcoming the user
 2. preparing the software for use and load existing data
 3. And, stuck at start?